src/mp/gen.cpp

```diff
@@ -630,7 +630,7 @@ int main(int argc, char** argv)
     } else {
         throw std::runtime_error(std::string("Failed to open src_prefix prefix directory: ") + argv[1]);
     }
-    for (size_t i = 4; i < argc; ++i) {
+    for (size_t i = 4; i < static_cast<size_t>(argc); ++i) {
         KJ_IF_MAYBE(dir, fs->getRoot().tryOpenSubdir(cwd.evalNative(argv[i]))) {
```

Comment:
In commit "Fix `-Wsign-compare` compiler warning" (7452ddd1ee11557de935af821782e9babc7036f5)

Is switching `size_t` on this line to `int` enough to fix the problem? It would seem better to avoid this error by using the same type consistently and avoiding the need to cast.

Reply:
re: https://github.com/chaincodelabs/libmultiprocess/pull/151#discussion_r1941803268

Went ahead and made this change in #152 but did not confirm it fixes the warning (since I wasn't seeing the warning). If not, can use the fix here instead.

Reply:
I can confirm that the [commit](https://github.com/chaincodelabs/libmultiprocess/pull/152/commits/0b8b7f92ca39d01f1422bfdbbbf55fb92857b7b9) from https://github.com/chaincodelabs/libmultiprocess/pull/152 fixed the warning.